packet/routes/api.py

```diff
@@ -0,0 +1,14 @@
+from packet import auth, app
+from packet.utils import before_request
+from packet.packet import sign
+
+
+@app.route("/api/v1/<member_username>/sign/<packet_username>")
+@auth.oidc_auth
+@before_request
+def sign(member_username, packet_username, uid):
```

Comment:
Is allowing `member_username` and `uid` to mismatch a planned feature (I can see the use for, e.g. the evals director)? If not, why allow it in the route at all?

Reply:
I did not think about the evals case.  I will make the change to recognize that case prior to merging

Reply:
It's also fine if that's not intended; I just don't see a reason to duplicate the signing member's username in the API call if it's always going to be the authenticated user.